src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java

```diff
@@ -96,7 +96,7 @@ private static void createTenantsTestTable() throws SQLException {
                 + "customer VARCHAR(50), " //
                 + "product VARCHAR(100), " //
                 + "quantity DECIMAL(18,0), " //
-                + "exa_row_tenants VARCHAR(128))");
+                + "exa_row_tenant VARCHAR(128))");
```

Comment:
Looking at this I think it would be a good idea to use uppercase all the way.

Reply:
Fixed in the IT test